Caroline/dictionary

cdebbaruah opened the pull request:
Completed dictionary styling for desktop and mobile screens!

DiyaBaliga commented on src/Pages/Dictionary.js:
```diff
@@ -1,6 +1,8 @@
+/* eslint-disable max-len */
```
THIS IS SO PRETTY OMG just have to get this off my chest first

DiyaBaliga commented on src/Pages/DictionaryPage.css:
```diff
@@ -0,0 +1,6 @@
+.DictionaryBackground {
```
lowercase + dashes

DiyaBaliga commented on src/Components/Dictionary/DictionaryCard.js:
```diff
@@ -1,65 +1,180 @@
             { linksArray.length !== 0 ? (
               <Grid
-                style={{ paddingLeft: 15 }}
                 sx={{
```
in-line styles

DiyaBaliga reviewed (changes requested):
CAROLINEEE!!! IT'S SO PRETTY OMG! I'm so impressed. I changed the margins so that the cards line up with the top text (check the most recent commits if you'd like to see how) but otherwise it's beautiful!!

There are a couple remaining in-line styles, so if you could go through and take them out that'd b v helpful! Otherwise it's perfect

cdebbaruah commented:
> CAROLINEEE!!! IT'S SO PRETTY OMG! I'm so impressed. I changed the margins so that the cards line up with the top text (check the most recent commits if you'd like to see how) but otherwise it's beautiful!!
> 
> There are a couple remaining in-line styles, so if you could go through and take them out that'd b v helpful! Otherwise it's perfect



> CAROLINEEE!!! IT'S SO PRETTY OMG! I'm so impressed. I changed the margins so that the cards line up with the top text (check the most recent commits if you'd like to see how) but otherwise it's beautiful!!
> 
> There are a couple remaining in-line styles, so if you could go through and take them out that'd b v helpful! Otherwise it's perfect

TY for reviewing my PR!! I've removed the in-line styles & renamed class attributes to be lowercase with dashes
